aexpect/session_ops.py

```diff
@@ -0,0 +1,598 @@
+# This Python file uses the following encoding: utf-8
```

Comment:
I usually try to avoid underscore on modules names, so a few suggestions:

`aexpect/session/ops.py`
`aexpect/session/utils.py`
`aexpect/session/commands.py`

Reply:
I am not sure where this convention originates from but in any case I would not create an entire directory structure for a single module. Names like `ops`, `utils` and `commands` are still good though.

Reply:
It is not "forbidden" by any PEP and yes you can have underscore, of corse. But I only use as last resource, when a single name cannot represent the module job.

From PEP8:

> Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.

I agree that having a package for a single module is too much, but if we think that we could split those commands into "categories"/modules, maybe will be needed in the future.

I'm ok if you decide to go without a new package.

Reply:
Yes, agreed. How about calling it the module `ops` here and in a later PR splitting it into `ops.<submodule>` packages? Does this sound ok with you?

Reply:
sounds good.